### PR TITLE
Rework SCALE Network Articles Into One

### PR DESCRIPTION
**Pull request by tonyriv3**
DOCS-2397 SCALE: Write Network introduction
DOCS-2398 SCALE: Interfaces article
DOCS-2399 SCALE: Static Routes article
DOCS-2400 SCALE: Global Configuration
DOCS-2401 SCALE: OpenVPN article
DOCS-2440 SCALE: IPMI article

### CODE REVIEW

**Comment by Mrt134** [comment]
@tonyriv3 the combined article looks good! I'm still concerned with the readability/verticality of this approach - can we rework the networking sections as tabs boxes instead of the headers and expandables? Sorry for being wishy-washy, but after seeing it in this form I think a tabs approach will end up being a little more reader-friendly.

**Comment by tonyriv3** [comment]
@Mrt134 Of course. I'll do that now.

**Comment by Mrt134** [comment]
I don't think they look too crowded, seems to be enough space :). Got one suggestion for a bit of transition text between intro and tabs box, adding that now.

File: content/SCALE/Network/_index.md [hunk 1]
```diff
@@ -4,10 +4,328 @@ geekdocCollapseSection: true
 weight: 60
 ---
 
+{{< toc >}}
+
 The SCALE Network section contains network configuration and settings options for active *Interfaces*, *Static Routes*, and the *Global Configuration*. 
 
 The Network screen also displays *OpenVPN* information and *IPMI* channels.
 
 ![NetworkSCALE](/images/SCALE/NetworkSCALE.png "SCALE Network Page")
```
Comment (Mrt134) [inline content/SCALE/Network/_index.md:14]: Rough suggestion to transition away from article intro to describing content in the tabs box.
```suggestion

Each Networking configurable is a separate widget on the overview screen. Click the buttons or an existing entry in a widget to slide out a side panel with configuration options. Select a networking section in the box below to see more details about the specific configuration options.
```

Comment (tonyriv3) [reply content/SCALE/Network/_index.md:14]: Done.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**